aviatrix/data_source_aviatrix_dcf_mwp_attachment_points_test.go

```diff
@@ -0,0 +1,56 @@
+package aviatrix
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccDataSourceAviatrixDcfMwpAttachmentPoints_basic(t *testing.T) {
+	resourceName := "data.aviatrix_dcf_mwp_attachment_points.test"
```

Comment:
The test doesn't pass atm.
The resource name has a typo:
```suggestion
	resourceName := "data.aviatrix_dcf_mwp_attachment_point.test"
```
Plus, the property `attachment_point_id` is not set:
```
=== RUN   TestAccDataSourceAviatrixDcfMwpAttachmentPoints_basic
time="2025-09-17T08:52:35Z" level=info msg="Getting API token..."
time="2025-09-17T08:52:35Z" level=info msg="Parsed Aviatrix login: admin"
time="2025-09-17T08:52:35Z" level=info msg="Getting API token..."
time="2025-09-17T08:52:35Z" level=info msg="Parsed Aviatrix login: admin"
time="2025-09-17T08:52:36Z" level=info msg="Getting API token..."
time="2025-09-17T08:52:36Z" level=info msg="Parsed Aviatrix login: admin"
    data_source_aviatrix_dcf_mwp_attachment_points_test.go:20: Step 1/1 error: Check failed: Check 3/3 error: data.aviatrix_dcf_mwp_attachment_point.test: Attribute 'attachment_point_id' expected to be set
time="2025-09-17T08:52:36Z" level=info msg="Getting API token..."
time="2025-09-17T08:52:36Z" level=info msg="Parsed Aviatrix login: admin"
--- FAIL: TestAccDataSourceAviatrixDcfMwpAttachmentPoints_basic (2.17s)
```
Only `"name"` and `"id"` are set.

Reply:
It would also be great if the test would clean up the controller after itself.
Otherwise there is a difference in behavior between running the test the first time and running it the second time.

Reply:
Added attachment_point_id to the test, the cleanup would not be required as this is not really a resource but just a mapping of name - uuid which should be consistent for multiple calls. So the behavior of the test will not be different if it is run for the second time, as the same GET call would return the same UUID for the same name.

Reply:
Discussed with @dnardo and it is not necessary to cleanup the attachment point uuid - name mapping from the DB as it would not be affecting any other tests that might run later on. And would be fine to leave it there without cleaning up.